components/src/preact/dateRangeSelector/date-range-selector.stories.tsx

```diff
@@ -23,7 +23,7 @@ const meta: Meta<DateRangeSelectorProps<'CustomDateRange'>> = {
     component: DateRangeSelector,
     parameters: {
         actions: {
-            handles: ['gs-date-range-changed'],
+            handles: ['gs-date-range-changed', previewHandles],
```

Comment:
```suggestion
            handles: ['gs-date-range-changed', ...previewHandles],
```
Also for all the other occurrences.